### PR TITLE
List component

### PR DESCRIPTION
**Pull request by utkarshducoder**
Please review my list components

this page contains the list components of Component library

live link: https://componentsofcss.netlify.app/component%20html/list.html

Following are the features of this page

nav: contains brand name and links to the home page and components library.
main: contains the documentation of list components with snippets and examples.
aside: contains links to all my component libraries.
footer: contains links to my social media

image

### CODE REVIEW

File: utkarsh component/list/list.css [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+    
+}
+
+a:hover{
+    color: #f472b6;
+  
+}
+
+main{
+
```
Comment (khushijohri2001) [inline utkarsh component/list/list.css]: Remove the unwanted code here. To make the code looks clean and readable

File: utkarsh component/list/list.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    
+        <script src="https://kit.fontawesome.com/e45a62e93a.js" crossorigin="anonymous"></script>
+
```
Comment (khushijohri2001) [inline utkarsh component/list/list.html:11]: remove extra space here

**Review by khushijohri2001 — APPROVED** [review]

File: utkarsh component/list/list.css [hunk 3]
```diff
@@ -0,0 +1,432 @@
...
+label.logo{
+  color: #fff;
+  font-size: 35px;
+  line-height: 80px;
+  padding: 0 100px;
+  font-weight: bold;
+}
+nav .main-list{
+  float: right;
+  margin-right: 20px;
```
Comment (i-am-a-codegirl) [inline utkarsh component/list/list.css:183]: avoid using pixel especially for lengths like padding, margin and font-size

File: utkarsh component/list/list.css [hunk 4]
```diff
@@ -0,0 +1,432 @@
...
+}
+#check{
+  display: none;
+}
+@media (max-width: 925px){
+  label.logo{
+    font-size: 30px;
+    padding-left: 50px;
+  }
+  nav  li a{
```
Comment (i-am-a-codegirl) [inline utkarsh component/list/list.css]: shift all the media queries in the end and not anywhere in between

**Review by i-am-a-codegirl — APPROVED** [review]

**Pull request merged**